libafl/src/executors/command.rs

```diff
@@ -494,6 +493,76 @@ where
     }
 }
 
+/// The shared implementation for children with stdout/stderr/timeouts.
+pub trait ChildrenArgs: Sized {
```

Comment:
I prefer `ChildArgs`

Also, maybe move it to executors since we need it in multiple places?

Reply:
At this moment I would prefer to keep it there as only `CommandExecutorBuilder` and `ForkserverExecutorBuilder` share it and I don't see other obvious components relying on this.